project/src/components/like-films/like-films.tsx

```diff
@@ -0,0 +1,25 @@
+import { useAppSelector } from '../../hooks';
+import { Film } from '../../types/film';
+import FilmList from '../film-list/film-list';
+import { chooseGenre } from '../generes-list/genres-list-functions';
+
+type LikeFilm = {
+  film: Film
+}
+function LikeFilms(props:LikeFilm) : JSX.Element {
+  const {film} = props;
+  const {genre} = film;
+  const films = useAppSelector((state) => state.films);
+  const currentFilms = chooseGenre(genre,films);
```

Comment:
Это надо делать внутри селектора
```suggestion
  const currentFilms = useAppSelector((state) => chooseGenre(genre,state.films));
```